go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -652,6 +657,7 @@ func (agent *ActionAgent) checkTabletMysqlPort(ctx context.Context, tablet *topo
 	}
 
 	if mport == topoproto.MysqlPort(tablet) {
+		agent.gotMysqlPort = true
```

Comment:
The function should be renamed `checkTabletMysqlPortLocked` because you assume that the lock on `actionMutex` is held?

Mention in the comment for the function which lock you're expecting to be held?